### PR TITLE
[add] papers

### PR DESCRIPTION
**Pull request by YashJain14**
StoryBench: A Multifaceted Benchmark for Continuous Story Visualization

Imagen Editor and EditBench: Advancing and Evaluating Text-Guided Image Inpainting

Davidsonian Scene Graph: Improving Reliability in Fine-grained Evaluation for Text-to-Image Generation

Benchmarking Spatial Relationships in Text-to-Image Generation

Rich Human Feedback for Text-to-Image Generation

Deep Reward Supervisions for Tuning Text-to-Image Diffusion Models

ByteEdit: Boost, Comply and Accelerate Generative Image Editing

A Contrastive Compositional Benchmark for Text-to-Image Synthesis: A Study with Unified Text-to-Image Fidelity Metrics

Confidence-aware Reward Optimization for Fine-tuning Text-to-Image Models

Directly Fine-Tuning Diffusion Models on Differentiable Rewards

Aligning Text-to-Image Diffusion Models with Reward Backpropagation

JourneyDB: A Benchmark for Generative Image Understanding

Stellar: Systematic Evaluation of Human-Centric Personalized Text-to-Image Methods

Navigating Text-To-Image Customization: From LyCORIS Fine-Tuning to Model Evaluation

ConceptBed: Evaluating Concept Learning Abilities of Text-to-Image Diffusion Models

GenAI-Bench: A Holistic Benchmark for Compositional Text-to-Visual Generation

Exploring GPT-4 Vision for Text-to-Image Synthesis Evaluation

Exploring AIGC Video Quality: A Focus on Visual Harmony, Video-Text Consistency and Domain Distribution Gap

Divide, Evaluate, and Refine: Evaluating and Improving Text-to-Image Alignment with Iterative VQA Feedback

Visual Programming for Text-to-Image Generation and Evaluation

LLMScore: Unveiling the Power of Large Language Models in Text-to-Image Synthesis Evaluation

X-IQE: eXplainable Image Quality Evaluation for Text-to-Image Generation with Visual Large Language Models

TIFA: Accurate and Interpretable Text-to-Image Faithfulness Evaluation with Question Answering

### CODE REVIEW

**Review by ziqihuangg — CHANGES_REQUESTED** [review]
Hi Yash, thanks for adding the related papers. Just a minor thing to fix: 

For the "Code link", it seems the first half of the link is pointing to a different repo. For example in line 140:
`  [Code](https://github.com/j-min/DSG)
`
the first link points to ImagenHub, while the second link points to DSG code. It would be great to fix this before merging. Thanks!
